app/workers/reload_queue_worker.rb

```diff
@@ -13,11 +13,14 @@ def perform(workflow_id)
     else
       reload_subjects
     end
+  rescue ActiveRecord::RecordNotFound
```

Comment:
Wondering here if a user:workflow queue had existed but the workflow was deleted before this event fired via sidekiq. If we should then cleanup any queues that may exist for that workflow?